### PR TITLE
Add View:  Make views generate name from collection.

### PR DESCRIPTION
**Pull request by kindly**
Also split out add_view into own function.

https://github.com/open-contracting/kingfisher-views/issues/65

### CODE REVIEW

File: docs/cli-add-view.rst [hunk 1]
```diff
@@ -5,18 +5,24 @@ This tool will add a new view.
...
 .. code-block:: shell-session
 
-    python ocdskingfisher-views-cli add-view viewname 1,3 "Created by Fred."
+    python ocdskingfisher-views-cli add-view 1,3 "Created by Fred."
 
-The view name must be something that is a valid Postgres schema name. We suggest using numbers, characters and _ only.
+This will create a schema called `view_data_collection_1_3`.  You can pass a single collection ID, or multiple collection ID's separated by commas.
 
-You can pass a single collection ID, or multiple collection ID's separated by commas.
+If you select more than 5 collection or if you want to choose your own view name you can pass a --name argument.  The view name must be something that is a valid Postgres schema name. We suggest using numbers, characters and _ only.
```
Comment (odscjames) [inline docs/cli-add-view.rst]: "If you select more than 5 collections" <- s

**Comment by odscjames** [comment]
Apart from tiny grammar thing, looks great, thanks!

File: docs/cli-add-view.rst (hunk 1 above)
Comment (kindly) [reply docs/cli-add-view.rst]: done

**Comment by kindly** [comment]
[Automate naming for views](https://trello.com/c/oyfNonw9/175-automate-naming-for-views)

**Pull request merged**